src/vellum/workflows/nodes/tests/test_mocks.py

```diff
@@ -277,3 +277,61 @@ class Outputs(BaseWorkflow.Outputs):
             raw_mock_with_invalid_key,
             MyWorkflow,
         )
+
+
+def test_mocks__node_not_found_in_workflow_skips_with_warning(caplog):
```

Comment:
DevinAI move this test to live with the other mock tests so that we have in `ee` so that we only consider the modern shape